whois-nrtm4/pom.xml

```diff
@@ -30,6 +30,10 @@
             <artifactId>whois-commons</artifactId>
             <type>test-jar</type>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.jetty</groupId>
+            <artifactId>jetty-rewrite</artifactId>
+        </dependency>
```

Comment:
This is alreday defined in parent pom, do we need to add again?

Reply:
er, that's not quite right. Miguel added it because it's needed by the Nrtm4Log class